db/migrate/20171012024407_create_votes.rb

```diff
@@ -0,0 +1,23 @@
+class CreateVotes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :votes do |t|
+      t.integer :vote
+      t.string :votable_type
+      t.integer :votable_id
+      t.integer :user_id, { null: false }
+      t.timestamps
+    end
+    add_index :votes, [:votable_type, :votable_id]
```

Comment:
Even better: add an index for `user_id` also

Reply:
FYI be aware that the order these fields are in matters. Check [this out](https://stackoverflow.com/a/6170023/5481524).